packages/loaders/json-schema/src/addExecutionLogicToComposer.ts

```diff
@@ -444,9 +444,11 @@ ${operationConfig.description || ''}
               }
               linkResolverFieldMap[linkName] = (root, args, context, info) =>
                 linkResolver(linkObj.args, targetField.resolve, root, args, context, info);
+              if (!linkObj.args) {
+              }
```

Comment:
can be removed

Reply:
https://github.com/Urigo/graphql-mesh/pull/4340/commits/ff7079b60af983dec775d43cef9bd3ea0ea71708
😅